src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -0,0 +1,1302 @@
+						for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+							tableBody( columnRecount - 9 + iBin, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeInBin[iBin], fldStIt->m_showDigits );
+						}
+						tableBody( columnRecount - 9 + iBin, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeInBinTotal[iBin], fldStIt->m_showDigits );
+					}
+					columnHead( columnRecount - 10 ) = fldStIt->m_colHead + curAggString + " LESS THAN BIN A";
+					for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+						tableBody( columnRecount - 10, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeBelowBottomBin, fldStIt->m_showDigits );
+					}
+					tableBody( columnRecount - 10, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeBelowBottomBinTotal, fldStIt->m_showDigits );
+					createBinRangeTable = true;
+				}
+				else if (curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsMinToZero ){
+					// put in the name of the variable for the column
+					if ( fldStIt->m_varAvgSum == isSum ) { // if it is a summed variable
+						curUnits += "/s";
+					}
+					fixUnitsPerSecond( curUnits, curConversionFactor );
+					for ( int iBin = 0; iBin != 10; iBin++ ){
+						char binIndicator = iBin + 65;
+						columnHead( columnRecount - 10 + iBin ) = fldStIt->m_colHead + curAggString + " BIN " + binIndicator;
+						for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+							tableBody( columnRecount - 10 + iBin, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeInBin[iBin], fldStIt->m_showDigits );
+						}
+						tableBody( columnRecount - 10 + iBin, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeInBinTotal[iBin], fldStIt->m_showDigits );
+					}
+					columnHead( columnRecount ) = fldStIt->m_colHead + curAggString + " MORE THAN BIN J";
+					for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+						tableBody( columnRecount , row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeAboveTopBin, fldStIt->m_showDigits );
+					}
+					tableBody( columnRecount, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeAboveTopBinTotal, fldStIt->m_showDigits );
+					createBinRangeTable = true;
+				}
+				else if ( curAgg == AnnualFieldSet::AggregationKind::hoursInTenPercentBins || curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsPlusMinusTwoStdDev ||
+					curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsPlusMinusThreeStdDev ){
+				}
+			} //fldStIt
+			OutputReportTabular::WriteReportHeaders( m_name, "Entire Facility", isAverage );
+			OutputReportTabular::WriteSubtitle( "Custom Annual Report" );
+			OutputReportTabular::WriteTable( tableBody, rowHead, columnHead, columnWidth, true ); //transpose annual XML tables.
```

Comment:
@mbadams5 I would like to make a unit test that somehow intercepts the file output stream that is used by OutputReportTabular:WriteTable in order to test if this routine is working.  I think I can follow the example of how that was done in EnergyPlusFixture. Should I add my code to that into EnergyPlusFiture or should I make a new fixture or just a new routine in OutputReportTabular?